### PR TITLE
Removing unnecessary strings from Crowdin + improving typos

### PR DESCRIPTION
**Pull request by kaktusus**
patches for #10459 and #10426...

a few small details escaped attention need to be fixed
:wink: 
Removing unnecessary strings from Crowdin and improving typos.

### CODE REVIEW

File: src/Mod/Fem/femcommands/commands.py [hunk 1]
```diff
@@ -210,7 +210,7 @@ def __init__(self):
         super(_ConstraintCentrif, self).__init__()
         self.menutext = Qt.QT_TRANSLATE_NOOP(
             "FEM_ConstraintCentrif",
-            "Centrifigual load"
+            "centrifugal load"
```
Comment (luzpaz) [inline src/Mod/Fem/femcommands/commands.py:213]: No capital 'C' here ?

Comment (kaktusus) [reply src/Mod/Fem/femcommands/commands.py:213]: damn

Comment (kaktusus) [reply src/Mod/Fem/femcommands/commands.py:213]: How do I correct it now?

Comment (luzpaz) [reply src/Mod/Fem/femcommands/commands.py:213]: Add the change
Then `git rebase -i  HEAD~2`
`fixup` the latest change
`git commit --amend -m "Removing unnecessary strings from Crowdin and improving typos" -m "Patches for https://github.com/FreeCAD/FreeCAD/pull/10459 and https://github.com/FreeCAD/FreeCAD/pull/10426"`
`git push --force`

**Pull request closed without merging**